rfcs/0003-xcmp-picasso-statemine.md

```diff
@@ -0,0 +1,252 @@
+
+# Overview
+
+This proposal sugget to open bidirectional HRMP channel between Picasso and Statemine. This will enable crosschain communication between Picasso and Statemine to enable various use cases including crosschain token transfer.
+
+## Steps
+
+0. Prepare
+1. Open channel request from Picasso to Statemine
+2. Statemine accept channel request and send request to Picasso
+3. Accept channel form Statemine to Picasso
+
+## Preparation
+
+One should have `Identity` on Kusama to create `Proposal` on  https://parachains.polkassembly.io/ . 
+
+Suggested amount is 50 KSM total for all operations on `Balance`.
+
+Picasso chain also should have Balance, better 22 KSM.
```

Comment:
Is this for accepting the channel? Which account on Picasso?

Reply:
will clarify,  it is upper bound to have Identi pallet entry for referenda proposal and sending XCM to Kusama. Upper bound.